inst/qmd_template/make_tag_push_summary.qmd

```diff
@@ -224,20 +226,23 @@ new_station_summary <- station_table(new_matches, "tag")
 reactable(new_station_summary)
 ```
 
-```{r}
+```{r, fig.cap="Figure 7. Number of newly-added detections per receiver. Points represent receiver locations, where colors reflect  the number of detections  and sizes  reflect  the number of fish at that location."}
+# Make another label for individual
+# I find mapview has limitations in legend and it cannot show the size values. Here I use leaflet  
 #| label: new-detection-map-leaflet
-#| fig-cap: Figure 7. Number of newly-added detections per receiver. Points represent receiver locations, while size and color reflect the number of detections at that location.
-#| eval: !expr "nrow(new_matches) > 0"
```

Comment:
Please add back this line; it's to make sure this chunk only runs if there are new matches

Reply:
it has already been added